src/cache/axios.ts

```diff
@@ -43,7 +43,7 @@ export interface CacheAxiosResponse<R = any, D = any> extends AxiosResponse<R, D
    *
    * @see https://axios-cache-interceptor.js.org/config/response-object#cached
    */
-  cached: boolean;
+  cached?: boolean;
```

Comment:
this will always be present

Reply:
Understood, but that is exactly what breaks the compatibility with axios 1.7.8